src/objects/core/models.py

```diff
@@ -40,6 +40,17 @@ def url(self):
         # zds_client.get_operation_url() can be used here but it increases HTTP overhead
         return f"{self.service.api_root}objecttypes/{self.uuid}"
 
+    @property
+    def version(self):
```

Comment:
Functionally I don't know how these models interact with each other but in the previous PR from Erhan I saw that [he set the version](https://github.com/maykinmedia/objects-api/pull/332/files#diff-6917b29ef1787115c2db3a76aabd3c9f3eb76dae6796598a38506a140fbaef26R61) (in a new model field) when the `clean` method was called. Would this be a alternative solution (and prevent unnecessary API calls)?